### ISSUE TITLE
Improve clarity of copyright licence descriptions

### ISSUE THREAD

**Issue by danhorst** [ndlib/curate_nd#219]
The copyright statements available to patrons are opaque. We should provide more guidance.

> Instead of listing current creative commons type licenses in dropdown change options to be in plain english instead.
>
> For example, instead of "Attribution-Noncommercial 3.0 United States" display something like "No commercial use of this content"

See [DLTP-141](https://jira.library.nd.edu/browse/DLTP-141) and additional text in [IDR-209](https://jira.library.nd.edu/browse/IDR-209).

---
[VecNet has an implementation of this](https://github.com/vecnet/vecnet-dl/blob/c5396032e19305d979c4ab6a38a224d5fe5c1dd5/app/views/curation_concern/_rights_modal.html.erb). We should base our solution on it.

Make sure that the list of licences is the same.

---
[Trello card](https://trello.com/c/11TF58mW)

**Comment by jeremyf** [comment]
We will take the intersection of VecNet's licenses and CurateND's licenses and use VecNet's description.

**Comment by dbrower** [comment]
The only license in CurateND and not in VecNet is the empty license "". Perhaps it should be removed?

Otherwise, it looks like the main difference between CurateND and VecNet is VN having a large button which triggers a bootstrap popup giving a short description for each license.
